### PR TITLE
fixes all files according to changes

### PR DESCRIPTION
**Pull request by iamdhruv21**

### CODE REVIEW

**Comment by iamdhruv21** [comment]
fixes routes

File: Core/Session.php [hunk 1]
```diff
@@ -8,6 +8,7 @@ public function destroy(): void
     {
         session_unset();
         session_destroy();
+        header('location: /');
```
Comment (iamarpitpatidar) [inline Core/Session.php]: a function should only do one tast at a time

File: resources/views/partials/show.php [hunk 2]
```diff
@@ -2,20 +2,29 @@
...
 use \Core\DB;
 
-if(!isset($_SESSION['logged']) || !$_SESSION['logged']){
+if(! auth()->check()){
     header("location: /");
     exit;
 }
 
-$id = $_GET['id'];
+DB::query('select * from mail where id = :id', [
```
Comment (iamarpitpatidar) [inline resources/views/partials/show.php]: don't fetch database inside views, controllers are supposed to be doing this

File: resources/views/show.view.php [hunk 3]
```diff
@@ -45,33 +48,36 @@
         </div>
         <div class="mail-list">
             <?php foreach ($result as $item) : ?>
-                <?php if ($item['receiver'] == $user['email']) : ?>
+<!--                --><?php //if ($item['username']."@anonmail.com" == $user['email']) : ?>
```
Comment (iamarpitpatidar) [inline resources/views/show.view.php]: don't use comments inside views

File: resources/views/show.view.php [hunk 4]
```diff
@@ -1,5 +1,8 @@
 <?php
-    require "partials/header.php";
+
+use Core\DB;
```
Comment (iamarpitpatidar) [inline resources/views/show.view.php]: don't import DB inside views

**Review by iamarpitpatidar — CHANGES_REQUESTED** [review]
make relevant changes

**Pull request merged**